lib/assesments/assessment_2/amna/fi_as2_amna_dashboard.dart

```diff
@@ -0,0 +1,84 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_interns/assesments/assessment_2/amna/screens/fi_as2_amna_add.dart';
+import 'package:flutter_interns/assesments/assessment_2/amna/screens/fi_as2_amna_notification.dart';
+import 'package:flutter_interns/assesments/assessment_2/amna/screens/fi_as2_amna_search.dart';
+import 'package:flutter_interns/assesments/assessment_2/amna/screens/home/fi_as2_amna_home.dart';
+import 'package:flutter_interns/assesments/assessment_2/amna/screens/profile/fi_as2_amna_profile.dart';
+import 'package:flutter_interns/assesments/assessment_2/amna/theme/fi_as2_amna_constants.dart';
+import 'package:flutter_svg/svg.dart';
+
+class FiAs2AmnaDashboard extends StatefulWidget {
+  const FiAs2AmnaDashboard({super.key});
+
+  @override
+  State<FiAs2AmnaDashboard> createState() => _F1As2AmnaDashboardState();
+}
+
+class _F1As2AmnaDashboardState extends State<FiAs2AmnaDashboard> {
```

Comment:
members defined at class level should start with a leading underscore
```_formKey```
so that they are encapsulated